src/transport.ts

```diff
@@ -139,16 +127,11 @@ export class WebRTCTransport implements Transport, Initializable {
   }
 
   private generateNoisePrologue(pc: RTCPeerConnection, ma: Multiaddr): Uint8Array {
-    let remoteCerthash = sdp.certhash(ma);
-    if (!remoteCerthash) {
-      throw inappropriateMultiaddr('no remote tls fingerprint in multiaddr');
-    }
-    let remote = base64.decode(remoteCerthash);
     if (pc.getConfiguration().certificates?.length === 0) {
       throw invalidArgument('no local certificate');
     }
-    let localCert = pc.getConfiguration().certificates![0];
-    if (localCert.getFingerprints().length === 0) {
+    let localCert = pc.getConfiguration().certificates?.at(0)!;
```

Comment:
OOOoooh I didn't know typescript allows optional chaining :o Need to learn more about it, thanks for showing me!